### DISCUSSION TITLE
Confirm to rewrite save file

### DISCUSSION THREAD

**Discussion by Branikolog** [ihhub/fheroes2#5603] [Ideas]
Later series of HoMM and the majority of other games always ask player confirmation on rewriting a save file. In my personal point of view such behaviour is an example of proper and friendly game design.
What do you think on removing experimental option and enabling it on default?
![image](https://user-images.githubusercontent.com/55348946/177758375-e9e3ed19-e361-4c77-b356-1a2820e99e42.png)

**Comment by zenseii** [comment]
I agree with this. I haven't looked at how the implementation works right now in that experimental option, but it sounds like a good change and a way to get rid of another experimental option.

**Reply by ihhub** [reply]
I agree with this too. This would prevent possible mistakes of overwriting save files.

**Comment by ihhub** [comment]
Hi @oleg-derevenetz , @idshibanov and @LeHerosInconnu what do you think?

**Reply by LeHerosInconnu** [reply]
Hello @ihhub,

I agree with this. :)

**Reply by oleg-derevenetz** [reply]
Hi @ihhub I agree.

**Reply by idshibanov** [reply]
Unanimous :)
